Clear documantion on the new release. (#112)
SpikeInterface/spikeinterface: doc/installation.rst

Installation
============

:code:`spikeinterface` is a Python package.

The actual "new API" (v0.90.0) is not release on pypi yet.
It will release
 in July 2021

> You have to do this to install `spikeinterface` afrom source:

Reviewer: ```suggestion
To use it now, you have to install `spikeinterface` from source:
```
